src/config/custom-config.js

```diff
@@ -49,7 +49,7 @@ module.exports = {
     onlyCategories: [
       'legal',
       'page-data',
-      'page-content',
+      // 'page-content',
```

Comment:
Should this be removed?

Reply:
@jeffchew That refers to a category that is commented out. Beacon would complain that `page-content` didn't exist so I commented it out for now. Not sure if it's safe to remove @IgnacioBecerra? Or is this something you were still working on. I missed it in the last PR review.

https://github.com/IBM/beacon-for-ibm-dotcom/blob/8bf51d72de6b7deb2a2bbcc0a89c7defecd8102c/src/config/custom-config.js#L200-L205

Reply:
It should be safe to remove, I think this one was a placeholder audit that was never created (as far as I know).